### PR TITLE
Simplify visit_handling.py

### PR DESCRIPTION
**Pull request by g-braeunlich**

### CODE REVIEW

File: python/desc/rubin_roman_sims/visit_handling.py [hunk 1]
```diff
@@ -1,25 +1,21 @@
...
-            if not amp_files:
-                ccds.append(det_name)
-        trimmed_ccd_lists[visit] = ccds
-    return trimmed_ccd_lists
-
+    return {
+        visit: [
+            det_name
+            for det_name in ccd_candidates
+            if not glob.glob(os.path.join(output_dir, "amp_*.fits*"))
```
Comment (jchiang87) [inline python/desc/rubin_roman_sims/visit_handling.py]: I think the glob pattern still needs to be formatted with the visit and det_name, i.e.,
```
            if not glob.glob(os.path.join(output_dir, f"amp_*{visit}*{det_name}*.fits*"))
```
There's some test code in the `tests` folder that exercises this.

**Review by jchiang87 — CHANGES_REQUESTED** [review]

File: python/desc/rubin_roman_sims/visit_handling.py (hunk 1 above)
Comment (g-braeunlich) [reply python/desc/rubin_roman_sims/visit_handling.py]: :scream: oh forgot that

Comment (g-braeunlich) [reply python/desc/rubin_roman_sims/visit_handling.py]: Should be corrected now

**Review by jchiang87 — APPROVED** [review]
looks good.

**Pull request merged**